src/playlist/Playlist.js

```diff
@@ -1,35 +1,87 @@
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
-import React from 'react';
+import React, { Component } from 'react';
 import {
+  Image,
+  Text,
   View,
-  ScrollView,
+  Dimensions,
 } from 'react-native';
+import ParallaxScrollView from 'react-native-parallax-scroll-view';
+import Icon from 'react-native-vector-icons/Ionicons';
+import { Actions } from 'react-native-router-flux';
+
 import * as actionCreators from './playlistActions';
 import { selectPodcast } from '../player/playerActions';
 import s from './playlistStyles';
 import PlaylistEntry from './PlaylistEntry';
 
-const Playlist = ({ playlist, actions }) => (
-  <View style={s.container}>
-    <View style={s.playlist}>
-      <ScrollView>
-        {playlist.map((podcast, index) =>
-          <PlaylistEntry
-            key={podcast.id}
-            podcast={podcast}
-            index={index}
-            playlist={playlist}
-            selectPodcast={actions.selectPodcast}
-          />
-        )}
-      </ScrollView>
-    </View>
-  </View>
-);
+const window = Dimensions.get('window');
+const PARALLAX_HEADER_HEIGHT = 280;
+const STICKY_HEADER_HEIGHT = 50;
+
+class Playlist extends Component {
+  renderStickyHeader() {
+    const { playlist } = this.props;
+    return (
+      <View style={s.stickySection}>
+        <Text style={s.stickySectionTitle}>{playlist.name}</Text>
+      </View>
+    );
+  }
+
+  renderBackground() {
+    const { playlist } = this.props;
+    const imageSource = {
+      uri: playlist.background,
+      width: window.width,
+      height: PARALLAX_HEADER_HEIGHT,
+    };
+
+    return (
+      <View key="background" style={s.background}>
+        <Image source={imageSource} />
+        <View style={s.backgroundOverlay} />
+      </View>
+    );
+  }
+
+  render() {
+    const { onScroll = () => {} } = this.props;
```

Comment:
This is a destructuring assignment with a default. I don't tink we need it unless we plan on passing in an onScroll to props to override the onScroll behavior.

See if we can remove this and the onScroll prop of the component.